verifiers/internal/gha/builder.go

```diff
@@ -385,11 +385,11 @@ func GetWorkflowInfoFromCertificate(cert *x509.Certificate) (*WorkflowIdentity,
 		return nil, err
 	}
 	if deprecatedSourceRepository != "" && sourceURI != "" &&
-		"https://github.com/"+deprecatedSourceRepository != sourceURI {
+		httpsGithubCom+deprecatedSourceRepository != sourceURI {
```

Comment:
are these changes part of this PR? Or it's a rebase issue?

Reply:
rebase issue, because the new linter version is blocking.